src/Core/Content/Cms/Events/CmsPageLoadedEvent.php

```diff
@@ -7,30 +7,30 @@
 use Shopware\Core\Framework\DataAbstractionLayer\EntityCollection;
 use Shopware\Core\Framework\Event\NestedEvent;
 use Shopware\Core\Framework\Event\ShopwareSalesChannelEvent;
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Core\System\SalesChannel\SalesChannelContext;
 use Symfony\Component\HttpFoundation\Request;
 
 #[Package('discovery')]
 class CmsPageLoadedEvent extends NestedEvent implements ShopwareSalesChannelEvent
 {
-    protected Request $request;
-
-    protected CmsPageCollection $result;
-
-    protected SalesChannelContext $salesChannelContext;
-
     /**
+     * @deprecated tag:v6.8.0 - $result type will be changed from EntityCollection to CmsPageCollection
+     *
      * @param CmsPageCollection $result
      */
     public function __construct(
-        Request $request,
-        EntityCollection $result,
-        SalesChannelContext $salesChannelContext
+        protected Request $request,
+        protected EntityCollection $result,
```

Comment:
BC checker is now complaining that the property changed from `CmsPageCollection` to `EntityCollection` but as this is widening, it should be okay :thinking: